constraints/regex/character-class-word-char.isl

```diff
@@ -0,0 +1,20 @@
+type::{
+  regex: "[A-Za-z0-9]",
+}
+valid::[
+  "A",
+  "M",
+  "Z",
+  "a",
+  "n",
+  "z",
+  "0",
+  "5",
+  "9",
+]
+invalid::[
+  " ",
+  ".",
+  "\t",
+]
```

Comment:
minor: I would add some invalid cases of "special" characters here like `# ~`

Reply:
added